geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreQueryTest.scala

```diff
@@ -68,6 +68,21 @@ class AccumuloDataStoreQueryTest extends Specification with TestWithMultipleSfts
       "and there are no results" >> { features.hasNext must beFalse }
     }
 
+    "process an exclude query correctly" in {
+      val fs = ds.getFeatureSource(defaultSft.getTypeName)
+
+      val query = new Query(defaultSft.getTypeName, Filter.EXCLUDE)
+
+      ds.getQueryPlan(query) must beEmpty
```

Comment:
@elahrvivaz two things:  1) thanks for reading my mind, and 2) Imma order hooced on fhonics now...